examples/python/easy_start_demo/demo.proto

```diff
@@ -0,0 +1,51 @@
+// 语法版本声明，必须放在非注释的第一行
+// Syntax version declaration, Must be placed on the first line of non-commentary
+syntax = "proto3";
+
+// 包名定义, Python中使用时可以省略不写(PS:我还要再Go中使用,所以留在这里了)
+// Package name definition, which can be omitted in Python (PS: I'll use it again in Go, so stay here)
+package demo;
+
+/*
+`message`是用来定义传输的数据的格式的, 等号后面的是字段编号
+消息定义中的每个字段都有唯一的编号
+总体格式类似于Python中定义一个类或者Golang中定义一个结构体
+*/
+/*
+`message` is used to define the structure of the data to be transmitted, After the equal sign is the field number.
+Each field in the message definition has a unique number.
+The overall format is similar to defining a class in Python or a structure in Golang.
+*/
+message Request {
+    int64 Cid = 1;
```

Comment:
As per the [protocol buffer style guide](https://developers.google.com/protocol-buffers/docs/style), message names should be `PascalCase`, but field names should be `snake_case`.

Reply:
ok